ovn/daemonset.sh

```diff
@@ -0,0 +1,335 @@
+#!/bin/bash
+#set -x
+
+#Always exit on errors
+set -e
+
+# The script renders j2 templates into yaml files in .${OVN_YAML_DIR}/
+
+# ensure j2 renderer installed
+## pip freeze | grep j2cli || pip install j2cli[yaml] --user
+export PATH=~/.local/bin:$PATH
+
+OVN_IMAGE=""
+OVN_IMAGE_PULL_POLICY=""
+OVN_NET_CIDR=""
+OVN_SVC_DIDR=""
+OVN_K8S_APISERVER=""
+OVN_GATEWAY_MODE=""
+OVN_GATEWAY_OPTS=""
+OVN_DB_REPLICAS=""
+OVN_MTU=""
+OVN_SSL_ENABLE=""
+KIND=""
+OVN_UNPRIVILEGED_MODE=""
+MASTER_LOGLEVEL=""
+NODE_LOGLEVEL=""
+DBCHECKER_LOGLEVEL=""
+OVN_LOGLEVEL_NORTHD=""
+OVN_LOGLEVEL_NB=""
+OVN_LOGLEVEL_SB=""
+OVN_LOGLEVEL_CONTROLLER=""
+OVN_LOGLEVEL_NBCTLD=""
+OVNKUBE_LOGFILE_MAXSIZE=""
+OVNKUBE_LOGFILE_MAXBACKUPS=""
+OVNKUBE_LOGFILE_MAXAGE=""
+OVN_MASTER_COUNT=""
+OVN_REMOTE_PROBE_INTERVAL=""
+OVN_HYBRID_OVERLAY_ENABLE=""
+OVN_DISABLE_SNAT_MULTIPLE_GWS=""
+OVN_MULTICAST_ENABLE=""
+OVN_EGRESSIP_ENABLE=
+OVN_V4_JOIN_SUBNET=""
+OVN_V6_JOIN_SUBNET=""
+
+# Parse parameters given as arguments to this script.
+while [ "$1" != "" ]; do
```

Comment:
```suggestion
while [ "$#" -gt 0 ]; do
```

Reply:
This is a copy-over from the ovn-kubernetes repository

I think we should not focus on improving those here, but we should instead try enhance in their repo, and use them from here as soon as we can.